packages/flutter_tools/lib/src/version.dart

```diff
@@ -73,14 +98,18 @@ class FlutterVersion {
 
   String _runGit(String command) => runSync(command.split(' '), workingDirectory: Cache.flutterRoot);
 
+  void ensureVersionFile() {
+    fs.file(fs.path.join(Cache.flutterRoot, 'version')).writeAsStringSync(_frameworkVersion);
+  }
+
   @override
   String toString() {
-    final String flutterText = 'Flutter • channel $channel • ${repositoryUrl == null ? 'unknown source' : repositoryUrl}';
+    final String flutterText = 'Flutter $_frameworkVersion • channel $channel • ${repositoryUrl == null ? 'unknown source' : repositoryUrl}';
```

Comment:
Create a separate `• version $_frameworkVersion` bullet.  Otherwise they might see "Flutter unknown", which would be confusing.

Reply:
i've special-cased the unknown version case to not display the string, so it just says "Flutter" in that case.